manifests/repository.pp

```diff
@@ -43,6 +43,9 @@
           'Fedora': {
             $location = "${url}Fedora_${osmajrelease}"
           }
+          'VirtuozzoLinux': {
```

Comment:
Given the URL is same as 'RedHat' these two cases can be merged as `'RedHat', 'VirtuozzoLinux'`

Reply:
cool. thanks. didn't know that is possible and didn't want to regex. I pushed a new patch